src/sage/knots/knotinfo.py

```diff
@@ -1309,6 +1326,37 @@ def is_oriented(self):
         """
         return not knotinfo_bool(self[self.items.unoriented])
 
+    @cached_method
+    def cosmetic_crossing_conjecture_verified(self):
+        r"""
+        Return whether the Cosmetic Crossing Conjecture has been verified
+        for ``self``.
+
+        From the KnotInfo `description page <https://knotinfo.math.indiana.edu/descriptions/cosmetic_crossing.html>`__:
+
+            A crossing change in a diagram of a knot ``K`` is called cosmetic if
+            the resulting diagram also represents ``K``. The cosmetic crossing
+            conjecture posits that for any knot ``K``, the only cosmetic crossing
+            changes are nugatory, i.e. there exists an embedded 2-sphere in
+            ``S3`` which intersects K only at the two points of the relevant
+            crossing. Conversely, it is not hard to see that any nugatory
+            crossing change is cosmetic.
+
+        EXAMPLES::
+
+            sage: knots = [K for K in KnotInfo if K.is_knot() and K.crossing_number() < 10]
+            sage: all(K.cosmetic_crossing_conjecture_verified() for K in knots)
+            True
+        """
+        cosmetic_crossing = self[self.items.cosmetic_crossing]
+        if self.crossing_number() == 0:
+            return True
+        if (not cosmetic_crossing or cosmetic_crossing == 'Unknown'):
+            return False
+        verified = not knotinfo_bool(cosmetic_crossing)
+        assert verified
+        return verified
```

Comment:
You should choose one or the other. It makes no sense to return `True` if passed as the `assert` will fail otherwise. So I would either do the return or raise an error (with no `return` statement).

(Technically, yes, there is a difference as we could turn off `assert` checks, but that isn't easy to do. Plus `assert`s are meant for things that should be true anyways.)